manifests/stage-ofed-driver/0010_ofed-driver-ds.yaml

```diff
@@ -78,7 +78,10 @@ spec:
               command:
                 [sh, -c, 'lsmod | grep mlx5_core']
             periodSeconds: 30
-            initialDelaySeconds: 30
+            # starting from  v1.21, Kubernetes doesn't use startupProbe during POD restarts
+            # to prevent crash loop after POD restarts, we should grant enough time to POD to
+            # fully boot before we start liveness checking
+            initialDelaySeconds: 570
```

Comment:
who did you come up with this number?

Reply:
We discussed this with @AbdYsn. We use the same delay as for `startupProbe` 10 minutes. If I remember correctly, @AbdYsn mentioned that choosing `startupProbe` delays is based on experiments on multipile different ENVs.

Reply:
@ykulazhenkov  can you point me to the kubernetes commit that change the behaviour?

Reply:
I think this PR changed behavior of startupProbe https://github.com/kubernetes/kubernetes/pull/98376

Reply:
I will ask community. Maybe this is some regression in Kuberntes, because I can't find any mentions that behavior of startupProbe should change.